akka-http-caching/src/main/mima-filters/10.1.8.backwards.excludes

```diff
@@ -0,0 +1,9 @@
+# Scala 2.13.0-M5
+ProblemFilters.exclude[MissingTypesProblem]("akka.http.caching.scaladsl.CachingSettings$")
```

Comment:
Probably because it's no companion any more? Good that we don't have to care for it, if it's only a problem for 2.13.0-M5.

Maybe we should disable MiMa for 2.13.0 for the time being, instead of having to had general excludes? WDYT, @raboof?

Reply:
Actually, it fails for me for all Scala versions with

```
[error]  * the type hierarchy of object akka.http.caching.scaladsl.CachingSettings is different in current version. Missing types {akka.http.impl.util.SettingsCompanionImpl}
```

which is strictly a binary incompatibility but actually the one we intended, so let's just replace the comment above.

Reply:
I agree it is probably better to disable MiMa for 2.13.0-M5 instead of creating filters (and potentially filtering things on 2.12 as well).

But not needed yet it looks like, since we want the filter for all versions in this case ;)